src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/InstancesController.cs

```diff
@@ -305,13 +263,13 @@ public async Task<ActionResult> Get(int instanceOwnerId, Guid instanceGuid)
         /// </summary>
         /// <param name="appId">the application id</param>
         /// <param name="instanceOwnerId">instance owner id</param>
+        /// <param name="instance">instance</param>
         /// <returns>instance object</returns>
         /// <!-- POST /instances?appId={appId}&instanceOwnerId={instanceOwnerId} -->
         [HttpPost]
-        [DisableFormValueModelBinding]
-        [Consumes("application/json", otherContentTypes: new string[] { "multipart/form-data" })]
+        [Consumes("application/json")]
         [Produces("application/json")]
-        public async Task<ActionResult> Post(string appId, int? instanceOwnerId)
+        public async Task<ActionResult> Post(string appId, int instanceOwnerId, [FromBody] Instance instance)
```

Comment:
I think it should be possible to use this method  with only the instance. it contains the instanceOwnerId.

Otherwise we need logic for handling duality between instanceOwnerId from both query param and body.

 Alt-1: with only instanceOwnerId, Alt-2 with instance template that contains the instance Owner Id . But not both at the same time.

Reply:
I'll remove it from the path. Then, the dualty of including it in the template or the route is handled by runtime, and that should be enough I think :)